File: DAC_API/Models/DTO/Authentication/AuthResponseDTO.cs
using DAC_API.Models.DTO.User;

namespace DAC_API.Models.DTO.Authentication {
    public class AuthResponseDTO {
        public bool IsSuccess { get; set; }
        public string Message { get; set; }
        public string Token { get; set; }
        public string RefreshToken { get; set; }
        public DateTime Expiration { get; set; }
        public UserResponseDTO User { get; set; }
    }
}

File: DailyAdventureChallenge/app/src/main/java/com/example/dailyadventurechallenge/data/dto/Authentication/AuthResponseDTO.kt
package com.example.dailyadventurechallenge.data.dto.Authentication

import com.example.dailyadventurechallenge.data.dto.User.UserResponseDTO

data class AuthResponseDTO(
    val isSuccess: Boolean,
    val message: String,
    val token: String,
    val refreshToken: String,
    val expiration: String,
    val user: UserResponseDTO
)
